apps/web/src/views/Farms/components/FarmCard/V3/FarmV3StakeAndUnStake.tsx

```diff
@@ -49,9 +48,10 @@ export const FarmV3LPTitle = ({
   title: string
   outOfRange: boolean
 }) => (
-  <StyledLink external href={liquidityUrl}>
```

Comment:
Should we remove the external here?

Reply:
I think we should keep it though. I prefer keeping user at where they are.

Reply:
Part of the LP position has been restored.

Reply:
I see. It's good to remove it then

Reply:
merged..lol

just remove it on https://github.com/pancakeswap/pancake-frontend/pull/6523